modules/api-explorer-client/public/index.html

```diff
@@ -9,7 +9,10 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Manifest is useless for api-explorer-client
```

Comment:
imo: `useless` is a strong word and not appropriate for public oss

```suggestion
      Manifest is unnecessary for api-explorer-client
```

Reply:
Thanks for suggesting 👍